app/javascript/components/payment-form.js

```diff
@@ -0,0 +1,82 @@
+import { u } from 'umbrellajs';
+import { loadScript } from '../utils';
+
+const FONT_SIZE = 16;
+
+function createCardElement(element, stripe) {
+  // Create an instance of Elements.
+  const elements = stripe.elements();
+
+  const style = {};
+
+  const classes = {};
+
+  // Create an instance of the card Element.
+  const card = elements.create('card', {classes, style});
+
+  card.mount(element);
+
+  return card;
+}
+
+// Handle real-time validation errors from the card Element.
+function addListenersToCard(card, errorContainer) {
+  card.addEventListener('change', function(event) {
+    if (event.error) {
+      errorContainer.textContent = event.error.message;
+    } else {
+      errorContainer.textContent = '';
+    }
+  });
+}
+
+// Submit the form with the token ID.
+function stripeTokenHandler(form, token) {
+  const input = u(form).find('.stripe-token-input');
+  input.attr('value', token.id);
+
+  // Submit the form
+  form.submit();
+}
+
+function addListenersToForm(form, card, errorContainer, stripe) {
+  form.addEventListener('submit', function(event) {
+    event.preventDefault();
+
+    stripe.createToken(card).then(function(result) {
+      if (result.error) {
+        // Inform the user if there was an error.
+        errorContainer.textContent = result.error.message;
+      } else {
+        // Send the token to your server.
+        stripeTokenHandler(form, result.token);
+      }
+    });
+  });
+}
+
+document.addEventListener("turbolinks:load", function() {
+  const $form = u('.payment-form');
```

Comment:
I am using umbrella js instead of jquery. It works the same but it is a bunch smaller and easier to import. You just use `u('.payment-form')` instead of `$('.payment-form')`